libminifi/include/core/ProcessContext.h

```diff
@@ -246,14 +238,14 @@ class ProcessContext : public controller::ControllerServiceLookup, public core::
       return initialized_;
   }
 
-  static constexpr char const* DefaultStateManagerProviderName = "defaultstatemanagerprovider";
+  static constexpr char const* DefaultStateStorageName = "defaultstatestorage";
```

Comment:
won't this affect backwards compatibility?

Reply:
I don't think so. It's just used as an ID (line 264 and 282), not compared with anything coming from the config. Well, it could ruin your config it you already had some other controller service with the ID of "defaultstatestorage", but chances are really slim and easy to fix.